pages/changelog.js

```diff
@@ -103,7 +103,7 @@ export default function Changelog() {
                     <p className="truncate text-lg font-medium">
                       {change.title}
                     </p>
-                    <div className="ml-2 flex flex-shrink-0">
+                    <div className="flex flex-shrink-0 ml-2">
```

Comment:
I think this change is not required?

```suggestion
                    <div className="ml-2 flex flex-shrink-0">
```